apps/aot/src/app/events/[year]/[day]/_components/challenge-layout.tsx

```diff
@@ -165,15 +165,25 @@ export function ChallengeLayout({
     const mouseDownHandler = (e: MouseEvent | TouchEvent) => {
       if (e instanceof MouseEvent) {
         // Get the current mouse position
-        isDesktop ? (x = e.clientX) : (y = e.clientY);
+        if (isDesktop) {
+          x = e.clientX;
+        } else {
+          y = e.clientY;
+        }
```

Comment:
[`@typescript-eslint/no-unused-expressions`](https://typescript-eslint.io/rules/no-unused-expressions): using a ternary as a shorthand `if`/`else` is a stylistic choice, but often trips folks up.